### ISSUE TITLE
elitetorrent-biz: Got redirected to https://wwvv.elitetorrent.nl/

### ISSUE THREAD

**Issue by ngosang** [Jackett/Jackett#8692]
```
Jackett.Common.IndexerException: Exception (elitetorrent-biz): Got redirected to another domain. Try changing the indexer URL to https://wwvv.elitetorrent.nl/.
 ---> System.Exception: Parse error
 ---> Jackett.Common.ExceptionWithConfigData: Got redirected to another domain. Try changing the indexer URL to https://wwvv.elitetorrent.nl/.
   at Jackett.Common.Indexers.CardigannIndexer.CheckIfLoginIsNeeded(WebClientStringResult Result, IHtmlDocument document) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/CardigannIndexer.cs:line 841
   at Jackett.Common.Indexers.CardigannIndexer.PerformQuery(TorznabQuery query) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/CardigannIndexer.cs:line 1355
   --- End of inner exception stack trace ---
   at Jackett.Common.Indexers.BaseWebIndexer.OnParseError(String results, Exception ex) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/BaseIndexer.cs:line 831
   at Jackett.Common.Indexers.CardigannIndexer.PerformQuery(TorznabQuery query) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/CardigannIndexer.cs:line 1684
   at Jackett.Common.Indexers.BaseIndexer.ResultsForQuery(TorznabQuery query) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/BaseIndexer.cs:line 347
   --- End of inner exception stack trace ---
   at Jackett.Common.Indexers.BaseIndexer.ResultsForQuery(TorznabQuery query) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/BaseIndexer.cs:line 347
   at Jackett.Common.Indexers.BaseWebIndexer.ResultsForQuery(TorznabQuery query) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/BaseIndexer.cs:line 808
   at Jackett.Server.Controllers.ResultsController.Results(ApiSearch requestt) in /home/vsts/work/1/s/src/Jackett.Server/Controllers/ResultsController.cs:line 224

Error on 20200517130549 for EliteTorrent.biz.txtJackett.Common.ExceptionWithConfigData: Got redirected to another domain. Try changing the indexer URL to https://wwvv.elitetorrent.nl/.
   at Jackett.Common.Indexers.CardigannIndexer.CheckIfLoginIsNeeded(WebClientStringResult Result, IHtmlDocument document) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/CardigannIndexer.cs:line 841
   at Jackett.Common.Indexers.CardigannIndexer.PerformQuery(TorznabQuery query) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/CardigannIndexer.cs:line 1355
```

**Comment by garfield69** [comment]
resolved by #8689

**Comment by ngosang** [comment]
Got redirected to another domain. Try changing the indexer URL to https://wvvw.elitetorrent.nl/
The domain is different, www
